packages/publish-app-layer/src/index.tsx

```diff
@@ -53,7 +61,11 @@ export default function () {
         devWindow.current = window.open("http://localhost:4005", "_blank");
       window.focus();
     }
-  }, [status]);
+    document.addEventListener('keydown', handleKeyPress);
+    return () => {
+      document.removeEventListener('keydown', handleKeyPress);
+    };
+  }, [status,handleKeyPress]);
```

Comment:
Please install the prettier plugin and fix the formatting. Missing space after comma.

Reply:
Done .